app/src/main/java/com/codesoom/assignment/interfaces/LoadingRepository.java

```diff
@@ -0,0 +1,51 @@
+package com.codesoom.assignment.interfaces;
+
+import com.codesoom.assignment.DefaultTask;
+
+import java.util.List;
+
+
+/**
+ * Task 엔티티 객체에 대한 목록 및 상세 조회
+ * <p>
+ * All Known Implementing Classes:
+ * TaskLoadingRepository
+ * </p>
+ */
+public interface LoadingRepository {
+    /**
+     * 모든 Task 객체를 List의 형태로 불러온다
+     * <p>
+     * Method Returns:
+     * List<Task> - Task 엔티티 객체를 내부 요소로 하는 List Collection 객체
+     * </p>
+     */
```

Comment:
```suggestion
    /**
     * 모든 Task를 리턴한다.
     * 
     * @return ,,,
     */
```

이런 형태가 되면 좋겠어요! 
javadoc에서 인자는 `@param <인자명> <설명>` 
리턴값은 `@return <설명>` 형태가 일반적이에요!

Reply:
감사합니다. 참고해서 수정해보았습니다. 전보다 훨씬 깔끔해져서 기분이 좋습니다 !